parsing/unit_info.mli

```diff
@@ -58,10 +58,24 @@ type t
     - the input source file is [dir/x.mli]
     - the module name is [Y]
     - the prefix is [target/y]
+
+    When calling, for example, [ocamlopt foo.ml._.preprocess] (where
+    foo.ml._.preprocess is a serialized, ppx-expanded AST of foo.ml), the "raw"
+    source file is foo.ml._.preprocess, while foo.ml is the "original" source
+    file.
```

Comment:
I think this comment doesn't really make sense here, since the earlier part of the comment saying what's in a `t` doesn't mention a distinction between "source file" and "raw source file". I would either rework also that earlier part of the comment, or just move this example to the comment on raw_source_file.